src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -1693,6 +1693,13 @@ public override BoundNode VisitFieldAccess(BoundFieldAccess node)
 
         private void VisitFieldAccessInternal(BoundExpression receiverOpt, FieldSymbol fieldSymbol)
         {
+            // VisitLvalue and VisitRValue don't accept null arguments, and if receiverOpt is null
+            // then there's nothing left to visit anyway.
+            if (receiverOpt == null)
```

Comment:
@AlekseyTs: so you mean throw a `if (node == null) { return; }` in the body for VisitLValue?  

VisitRValue is safe to call with null, but it looks like it was almost by accident.

If I do change VisitLValue and VisitRValue to return immediately on null, should I change the argument name to nodeOpt?